cmd/kube-proxy/app/server_linux.go

```diff
@@ -91,13 +91,14 @@ func (s *ProxyServer) platformCheckSupported(ctx context.Context) (ipv4Supported
 	if isIPTablesBased(s.Config.Mode) {
 		// Check for the iptables and ip6tables binaries.
 		var ipts map[v1.IPFamily]utiliptables.Interface
-		ipts, err = utiliptables.NewDualStack()
+		var errDS error
+		ipts, errDS = utiliptables.NewDualStack()
 
 		ipv4Supported = ipts[v1.IPv4Protocol] != nil
 		ipv6Supported = ipts[v1.IPv6Protocol] != nil
 
 		if !ipv4Supported && !ipv6Supported {
-			err = fmt.Errorf("iptables is not available on this host : %w", err)
+			err = fmt.Errorf("iptables is not available on this host : %w", errDS)
 		} else if !ipv4Supported {
 			logger.Info("No iptables support for family", "ipFamily", v1.IPv4Protocol, "error", err)
```

Comment:
```suggestion
			logger.Info("No iptables support for family", "ipFamily", v1.IPv4Protocol, "error", errDS)
```

Reply:
same in the log below, 

```
			logger.Info("No iptables support for family", "ipFamily", v1.IPv6Protocol, "error", errDS)
```

Reply:
Can you squash the changes together? Just fix it all locally in one commit on top of master and then `git push --force`

Reply:
yes, i've squashed all the commits into one